src/AccessibilityInsights.Extensions.AzureDevOps/FileIssue/FileIssueHelpers.cs

```diff
@@ -240,15 +241,23 @@ internal static string RemoveInternalHTML(string inputHTML, string keyText)
 
             if (node != null)
             {
-                foreach (var n in node.childNodes)
+                foreach (IHTMLDOMNode child in GetChildren(node).ToList())
                 {
-                    node.removeChild(n);
```

Comment:
The old code was modifying a list while traversing it, which can lead to problems. The new code traverses a copy of the list so there's no interference